SPECS-EXTENDED/python-pyroute2/python-pyroute2.spec

```diff
@@ -0,0 +1,233 @@
+%global srcname pyroute2
+
+%{!?python3_pkgversion:%global python3_pkgversion 3}
+
+# FIXME(hguemar): Fix for EL7, in summary
+# Fedora => python3 only
+# EL7 => python2 only
+# EL>7 => python3 only
+
+Name: python-%{srcname}
+Version: 0.5.14
+Release: 4%{?dist}
+Summary: Pure Python netlink library
+License: GPLv2+
+URL: https://github.com/svinota/%{srcname}
+
+BuildArch: noarch
+Source0: https://pypi.io/packages/source/p/pyroute2/pyroute2-%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+%description
+PyRoute2 provides several levels of API to work with Netlink
+protocols, such as Generic Netlink, RTNL, TaskStats, NFNetlink,
+IPQ.
+
+
+%package -n python%{python3_pkgversion}-%{srcname}
+Summary: %{summary}
+BuildRequires: python%{python3_pkgversion}-devel
+BuildRequires: python%{python3_pkgversion}-setuptools
+%{?python_provide:%python_provide python%{python3_pkgversion}-%{srcname}}
+
+%description -n python%{python3_pkgversion}-%{srcname}
+PyRoute2 provides several levels of API to work with Netlink
+protocols, such as Generic Netlink, RTNL, TaskStats, NFNetlink,
+IPQ.
+
+
+%prep
+%setup -q -n %{srcname}-%{version}
+cat README.rst | python3 ./docs/conv.py >README.md
+
+%build
+python3 setup.py build '--executable=/usr/bin/python3 -s'
+
+%install
+python3 setup.py install -O1 --skip-build --root %{buildroot}
+
+%files -n python%{python3_pkgversion}-%{srcname}
+%{_bindir}/ss2
+%{_bindir}/%{srcname}-cli
+%doc README* LICENSE.GPL.v2 LICENSE.Apache.v2
+%{python3_sitelib}/%{srcname}*
+
+%changelog
+* Wed Jan 5 2022 Cameron Baird <cameronbaird@microsoft.com>  - 0.5.14-4
+- Add to SPECS-EXTENDED from Fedora
```

Comment:
Please use the standard `Initial CBL-Mariner import from Fedora 35 (license: MIT)` line here